### ISSUE TITLE
Quick start Wizard error

### ISSUE THREAD

**Issue by nassl70** [texstudio-org/texstudio#3335]
### Environment
- TeXstudio:  4.6.3
- Qt:  6.5.2
- OS:  win11
- TeX distribution:  miktex

Quick start Wizard generates an incorrect entry in the preamble the geometry package when I use Russian language in the Options -> Configure TexStudio -> General -> Language -> ru_Ru.   Indents are set in Russian units (мм or см) instead of Englis
![изображение_2023-10-18_220709067](https://github.com/texstudio-org/texstudio/assets/148343526/ce1b90ca-797d-4505-b922-4351406fc49f)
h ones (mm or cm). This causes a compilation error.

### Expected behavior
\usepackage[left=2cm, right=2cm, top=2cm, bottom=2cm]{geometry}

### Actual behavior

\usepackage[left=2см, right=2см, top=2см, bottom=2см]{geometry}

**Comment by octaeder** [comment]
so like this:

![grafik](https://github.com/texstudio-org/texstudio/assets/102688820/fd24aea9-9f09-41c6-8e35-13ba7da2066f)

I make a PR.

**Comment by nassl70** [comment]
OK, but мм and см - by default now 
--
Отправлено из Mail.ru для Android среда, 18 октября 2023г., 22:37 +10:00 от octaeder  ***@***.*** :

**Comment by octaeder** [comment]
End texstudio, then delete following entries in %appdata%\texstudio\texstudio.ini (just in case make a backup of the file)
```
Quick\Geometry%20Page%20Width%20Unit=cm
Quick\Geometry%20Page%20Height%20Unit=cm
Quick\Geometry%20Margin%20Left%20Unit=cm
Quick\Geometry%20Margin%20Right%20Unit=cm
Quick\Geometry%20Margin%20Top%20Unit=cm
Quick\Geometry%20Margin%20Bottom%20Unit=cm
```

**Comment by nassl70** [comment]
This script does not change the behavior of the wizard. 
The units in the drop-down menu remain in Russian.
Yes, this is not difficult to fix in the final latex document, but the generator clearly needs to fix the logic

**Comment by nassl70** [comment]
The problem is fixed in version TeXstudio 4.7.0. Thanks! 
But why did the file encoding line disappear from the wizard? 
Like this:      \usepackage[utf8]{inputenc}

**Comment by nassl70** [comment]
> This is no longer necessary. See discussion of the change in #2831 (search for UTF8).

Its funny, I didn't know about this for utf and T1. Thanks!
